source/includes/_errors.md

```diff
@@ -14,6 +24,7 @@ Error Code | Meaning
 410 | Gone -- The kitten requested has been removed from our servers
 413 | Request Entity Too Large -- You sent too much data to us
 418 | I'm a teapot
+422 | Unprocessable Entity
```

Comment:
It's still heavy on a client *error*, which isn't the case here. I'll change it for consistency.